GAME/TYPES.H

```diff
@@ -14,7 +14,15 @@
 #endif
 
 #ifndef NULL
-#define NULL 0
+#define NULL ((void*)0)
```

Comment:
It is a good idea but there's headers already included in the project which define NULL so this seems redundant.